extension/src/popup/basics/Buttons.tsx

```diff
@@ -94,8 +95,12 @@ const BackButtonEl = styled(BasicButton)`
   }
 `;
 
-export const BackButton = ({ onClick, ...props }: BackButtonProps) => (
-  <BackButtonEl onClick={onClick} {...props}>
-    <Icon.ArrowLeft /> BACK
+export const BackButton = ({
+  onClick,
+  isPopup = false,
+  ...props
+}: BackButtonProps) => (
+  <BackButtonEl onClick={onClick} isPopup={isPopup} {...props}>
+    <Icon.ArrowLeft /> {!isPopup && "BACK"}
   </BackButtonEl>
```

Comment:
This is fine for this iteration. I'm working on some better reusable components, including Back buttons, etc. that we can use when we start adding styles